README.md

```diff
@@ -18,7 +25,7 @@ Treeherder exists on two instances, [stage](https://treeherder.allizom.org) for
 #### Installation
 The steps to run Treeherder are provided [here](https://treeherder.readthedocs.io/installation.html).
 
-The steps to run only the UI are provided [here](https://treeherder.readthedocs.io/ui/installation.html).
+The steps to run only the UI is provided [here](https://treeherder.readthedocs.io/ui/installation.html).
```

Comment:
I'm pretty sure that "are" is correct in this context? (Since "steps" is plural)